Cargo.toml

```diff
@@ -71,6 +71,8 @@ fast-rng = ["rng", "rand"]
 sha1 = ["sha1_smol"]
 md5 = ["md-5"]
 
+borsh = ["dep:borsh", "dep:borsh-derive"]
```

Comment:
The fact that this works on our MSRV now is great :+1: There's so much I can clean up elsewhere with our dependencies.